src/openrct2/scripting/ScCheats.hpp

```diff
@@ -0,0 +1,326 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2020 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#ifdef ENABLE_SCRIPTING
+
+#    include "../Cheats.h"
+#    include "Duktape.hpp"
+#    include "ScriptEngine.h"
+
+namespace OpenRCT2::Scripting
```

Comment:
Why not?
```suggestion
using OpenRCT2::Scripting;
```

Then you won't have to open a huge namespace and with the extra 4 spaces maybe somethings won't break in as many lines hahaha

Reply:
You shouldn't do this in header files. Unless scoped which this is not.

Reply:
Ah, I hadn't noticed it was a header